### DISCUSSION TITLE
Need help in training for telugu language

### DISCUSSION THREAD

**Discussion by shashidhar2609** [coqui-ai/TTS#666] [General Q&A]
Hi,

I have been trained telugu text to speech using keithio  tacotron previously by converting telugu text to transliterated english text ,it was working good but for longer sentences it was not performing well.i found your model aligns good for even longer sentences.so i was trying to train using ljspeech-ddc .i have 30 hours of single speaker data and corresponding transliterated text. even though model trained for 17000 steps the output of synthesis.py is null. could you please help by letting me know where i am going wrong.i have set  "use_phonemes": false.

command :python synthesize.py --text "anndaatku anni veelllloo andubaattuloo unddeenduku raassttr prbhutvn" --config_path /root/tacotron/LJSpeech/ljspeech-ddc-July-14-2021_08+44PM-e9e0784/config.json --model_path /root/tacotron/LJSpeech/ljspeech-ddc-July-14-2021_08+44PM-e9e0784/checkpoint_14000.pth.tar --out_path ./predicted
2021-07-15 10:10:16.604787: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
 > Setting up Audio Processor...
 | > sample_rate:22050
 | > resample:False
 | > num_mels:80
 | > min_level_db:-100
 | > frame_shift_ms:None
 | > frame_length_ms:None
 | > ref_level_db:20
 | > fft_size:1024
 | > power:1.5
 | > preemphasis:0.0
 | > griffin_lim_iters:60
 | > signal_norm:True
 | > symmetric_norm:True
 | > mel_fmin:50.0
 | > mel_fmax:8000.0
 | > spec_gain:20.0
 | > stft_pad_mode:reflect
 | > max_norm:4.0
 | > clip_norm:True
 | > do_trim_silence:True
 | > trim_db:60
 | > do_sound_norm:False
 | > stats_path:None
 | > hop_length:256
 | > win_length:1024
 > Using model: Tacotron2
 > Text: anndaatku anni veelllloo andubaattuloo unddeenduku raassttr prbhutvn
 > Text splitted to sentences.
['anndaatku anni veelllloo andubaattuloo unddeenduku raassttr prbhutvn']
 > Processing time: 0.25736236572265625
 > Real-time factor: 0.4612191290787199
 > Saving output to ./predicted/anndaatku_anni_veell.wav

the output anndaatku_anni_veell.wav has nothing in it.

training paramteres:
12|tts_train  |    --> STEP: 654/850 -- GLOBAL_STEP: 17675
12|tts_train  |      | > decoder_loss: 0.49871  (0.48944)
12|tts_train  |      | > postnet_loss: 0.55522  (0.55548)
12|tts_train  |      | > stopnet_loss: 2.01851  (2.04752)
12|tts_train  |      | > decoder_coarse_loss: 0.73749  (0.70555)
12|tts_train  |      | > decoder_ddc_loss: 0.01430  (0.02146)
12|tts_train  |      | > ga_loss: 0.00012  (0.00019)
12|tts_train  |      | > decoder_diff_spec_loss: 0.24318  (0.24266)
12|tts_train  |      | > postnet_diff_spec_loss: 0.32557  (0.32943)
12|tts_train  |      | > decoder_ssim_loss: 0.56306  (0.50855)
12|tts_train  |      | > postnet_ssim_loss: 0.58082  (0.52771)
12|tts_train  |      | > loss: 1.19283  (1.15014)
12|tts_train  |      | > align_error: 0.52511  (0.51386)
12|tts_train  |      | > max_spec_length: 601.0
12|tts_train  |      | > max_text_length: 112.0
12|tts_train  |      | > step_time: 3.2762
12|tts_train  |      | > loader_time: 0.01
12|tts_train  |      | > current_lr: 0.0001

Thanks

**Comment by rajashekhargithubb** [comment]
Hello did you found anything?

**Comment by RAVINDRA8008** [comment]
bro got any solution???  i need i am doing a project so?
